core/src/test/scala/org/http4s/rho/bits/HListToFuncSpec.scala

```diff
@@ -25,5 +25,14 @@ class HListToFuncSpec extends Specification {
       val req = Get("/route1")
       checkOk(req) should_== "foo"
     }
+
+    // Tests issue 218 https://github.com/http4s/rho/issues/218
+    "Work with cats.implicits" in {
+      import cats.implicits._
+      new RhoService[IO] {
+        GET / "route1" |>> { () => Ok("foo".pure[IO]) }
```

Comment:
Can we add a comment to say that the `pure[IO]` is here just for the import? My concern is people thinking that this is somewhat necessary.

Reply:
Done.

Reply:
thanks :D